src/streams/StreamScreen.js

```diff
@@ -63,10 +63,10 @@ class StreamScreen extends PureComponent<Props> {
     dispatch(toggleStreamNotification(stream.stream_id, !currentValue));
```

Comment:
Please squash this commit with its parent, to maintain the invariant that all commits pass all tests.

I was really excited when I first learned about the tricks for tidying commits on a PR, from [this doc](https://zulip.readthedocs.io/en/latest/git/fixing-commits.html). That doc is really the key piece for enabling the standard of [clear and coherent commits](https://zulip.readthedocs.io/en/latest/contributing/version-control.html); without it, I'd be stuck, feeling like I just had one shot at making good commits. It's tricky at first, but with practice, it becomes second-nature. 🙂

Reply:
Sure will do it. Haha squashing commits definitely helps tidy the huge commit hell xD